notebooks/01-visualize-amounts-bar-plots.ipynb

```diff
@@ -0,0 +1,1384 @@
+{

```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    bar_width = 0.2
</pre><p> </p><p>comment is redundant (the property name <code>bar_width</code> already says just as much) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokeh/dataviz-fundamentals/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>